engine/src/fixture.cpp

```diff
@@ -1428,7 +1425,6 @@ bool Fixture::saveXML(QXmlStreamWriter *doc) const
         doc->writeTextElement(KXMLFixtureForcedLTP, list);
     }
 
-    if (m_channelModifiers.isEmpty() == false)
```

Comment:
no need to check if the map is empty. If it is empty, the loop below will have zero iterations, so de facto isEmpty check is being made by it.hasNext()